ebpf-profiling.graphqls

```diff
@@ -152,13 +171,18 @@ enum EBPFProfilingAnalyzeAggregateType {
 extend type Mutation {
     # create a new eBPF fixed time profiling task
     createEBPFProfilingFixedTimeTask(request: EBPFProfilingTaskFixedTimeCreationRequest!): EBPFProfilingTaskCreationResult!
+
+    # create a new eBPF network profiling task
+    createEBPFNetworkProfiling(request: EBPFProfilingNetworkTaskRequest!): EBPFProfilingTaskCreationResult!
+    # keep alive the eBPF profiling task
+    keepAliveEBPFProfilingTask(taskId: ID!): EBPFProfilingTaskKeepAliveResult!
```

Comment:
```suggestion
    keepEBPFNetworkProfiling(taskId: ID!): EBPFProfilingTaskKeepAliveResult!
```
I think this is better? Could you change the return type name as well?

Reply:
Update to `EBPFNetworkProfilingKeepResult`. Is this better?